contract_price_revision/__manifest__.py

```diff
@@ -6,12 +6,12 @@
 {
     "name": "Contract Price Revision",
     "summary": "Easy revision of contract prices",
-    "version": "17.0.1.0.0",
+    "version": "18.0.1.0.0",
     "category": "Contract",
     "author": "ACSONE SA/NV, Tecnativa, Odoo Community Association (OCA)",
     "license": "AGPL-3",
     "website": "https://github.com/OCA/contract",
-    "depends": ["contract"],
+    "depends": ["contract_line_successor"],
```

Comment:
This dependency shouldn't be changed. This module should be independent from the other one, not setting that successor fields (they are only informative anyway).

Reply:
This dependency is necessary because the `predecessor_contract_line_id` field (used in https://github.com/OCA/contract/blob/d1709eddc7dfca0123e81f71433f0e94a6f2b698/contract_price_revision/models/contract_line.py# L13) is defined by the `contract_line_successor` module.

Reply:
Uhm, I don't like too much to have it, but OK.